Corona-Warn-App/src/main/res/values-night/colors.xml

```diff
@@ -23,7 +23,7 @@
     <color name="colorTextEmphasizedButton">#FFFFFF</color>
     <color name="colorTextSixteen">#FFFFFF</color>
     <color name="colorInputText">#BFFFFFFF</color>
-    <color name="colorTextSemanticRed">#FF919A</color>
+    <color name="colorTextSemanticRed">#ED5360</color>
```

Comment:
There are other UI elements that use this color apart from these red delete buttons. Are you sure they should have this color changed in dark mode or should it be restricted to only the dialog texts?

Reply:
Added color colorTextDeleteButtonDialog for dialogs